README.md

```diff
@@ -69,9 +69,27 @@ Infoblox Provider is a simple web server with several clearly defined routers:
 | /adjustendpoints | POST   |
 
 #### Reading Data
+Read data by HTTP GET to `/records`, see:
 ```shell
 curl -H 'Accept: application/external.dns.webhook+json;version=1' localhost:8888/records
 ```
+If you set DOMAIN_FILTER, DNS will return all records from this domain(s). Because the returned data for a given
+domain can be robust - in some cases tens of thousands of records, it is advisable to use filters to reduce the 
```

Comment:
```suggestion
domain can be large - in some cases tens of thousands of records, it is advisable to use filters to reduce the 
```

Robust doesn't seem like the best word 😄

Reply:
thx commited